ci/tests/e2e.sh

```diff
@@ -128,7 +125,10 @@ kubectl config set-context heptio-sonobuoy --cluster kubernetes --user admin --n
 kubectl config use-context heptio-sonobuoy
 
 # These files should have been included in a docker build
-kubectl apply -f /tmp/ci/sonobuoy/sonobuoy.yaml
+sonobuoy run
```

Comment:
It's not all there yet... For now, we can just crib the .yaml from the old release here.

Reply:
https://raw.githubusercontent.com/heptio/sonobuoy/release-0.10/examples/quickstart.yaml

Reply:
Or just point to the tag'd version ^